Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -105,6 +105,23 @@ public extension Sequence {
     func filtered<T>(_ isIncluded: (Element) throws -> Bool, map transform: (Element) throws -> T) rethrows -> [T] {
         return try lazy.filter(isIncluded).map(transform)
     }
+    
+    /// SwifterSwift: Contains operations based on matching function
+    ///
+    ///      ["James", "Wade", "Bryant"].contains([4,5], for: { $0.count == $1 }) -> true
```

Comment:
I know there are other convenience functions like that already here but I think this is something that can be already achieved by `["James", "Wade", "Bryant"].lazy.map(\.count).contains([4,5])` so I don't see a big win with this one. But if others are fine with including this I don't see a problem.

Reply:
I'll wait for input then to see what others think.

Reply:
I don't really see the purpose of this function. I read the example as "the array of names contains an array of numbers for a given comparison between the length of the names and each number". It would be very unclear to read this code in a real-world project. The example given by Luciano, however, is much clearer and just as simple to write.